--- node_modules/react-instantsearch-hooks-server/dist/es/getServerState.js	2022-09-12 13:14:07.237878664 -0700
+++ node_modules/react-instantsearch-hooks-server/dist/es-new/getServerState.js	2022-09-12 11:46:17.394188509 -0700
@@ -151,21 +151,5 @@
 }

 function importRenderToString() {
-  // React pre-18 doesn't use `exports` in package.json, requiring a fully resolved path
-  // Thus, only one of these imports is correct
-  var modules = ['react-dom/server.js', 'react-dom/server']; // import is an expression to make sure https://github.com/webpack/webpack/issues/13865 does not kick in
-
-  return Promise.all(modules.map(function (mod) {
-    return import(mod).catch(function () {});
-  })).then(function (imports) {
-    var ReactDOMServer = imports.find(function (mod) {
-      return mod !== undefined;
-    });
-
-    if (!ReactDOMServer) {
-      throw new Error('Could not import ReactDOMServer.');
-    }
-
-    return ReactDOMServer.renderToString;
-  });
-}
\ No newline at end of file
+  return import('react-dom/server.js').then((ReactDOMServer) => ReactDOMServer.renderToString);
+}
